packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -725,11 +739,11 @@ class ListTile extends StatelessWidget {
       ?? tileTheme.contentPadding?.resolve(textDirection)
       ?? defaultContentPadding;
 
-    final Set<MaterialState> states = <MaterialState>{
-      if (!enabled || (onTap == null && onLongPress == null)) MaterialState.disabled,
-      if (selected) MaterialState.selected,
+    // Update disabled state for mouse cursor so the cursor isn't updated when ListTile
+    // is enabled and selected but `onTap` and `onLongPress` callbacks are null.
+    states = <MaterialState>{
```

Comment:
Can' this just use the existing `states` from above? It may have a selected state, but that should be ok for the resolve of the cursor.